ui-data/html/analysis.html

```diff
@@ -128,6 +138,8 @@
                 foundItemRow.querySelector('input.signaloffset').value = parsedJson.so;
                 foundItemRow.querySelector('input.issigned').checked = parsedJson.s == true;
                 foundItemRow.querySelector('input.littleendian').checked = parsedJson.bo == true;
+                foundItemRow.querySelector('input.drivelog').checked = parsedJson.dl == true;
```

Comment:
Depending on how we handle these new properties we will need to update this code to deal with their existence in the parsed json